charts/em-npa/values.template.yaml

```diff
@@ -0,0 +1,73 @@
+postgresql:
+#  initdbScripts:
+#    init.sql: |-
+#      CREATE USER emnpa WITH PASSWORD 'emnpa';
+#      CREATE DATABASE emnpa WITH OWNER = emnpa ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+  postgresqlUsername: emnpa
+  postgresqlPassword: emnpa
+  postgresqlDatabase: emnpa
+  nameOverride: rpa-native-pdf-annotator-db
+  persistence:
+    enabled: false
+  tags:
+    postgresql-pod: true
+
+npa-app:
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  applicationPort: 8080
+  keyVaults:
+    em-npa:
+      resourceGroup: em-npa
+      # Pawel, what secrets do we need in here, and what should they be called?
+      secrets:
+#        - APPINSIGHTS_INSTRUMENTATIONKEY
+        # FUNCTIONAL_TEST_CLIENT_S2S_TOKEN or microservicekey-em-gw?
```

Comment:
Values in here should be key vault secret key names, not env vars. Env vars can be mapped in bootstrap.yaml